jsonnet/components/kube-state-metrics.libsonnet

```diff
@@ -266,6 +266,38 @@ function(params)
                           readOnly: true,
                         },
                       ],
+                      local metricsPort = 8081,
+                      local selfPort = 8082,
+                      ports::: [
```

Comment:
`:::` since [kube-prometheus hides these fields](https://github.com/prometheus-operator/kube-prometheus/blob/main/jsonnet/kube-prometheus/components/kube-state-metrics.libsonnet#L164-L166).